src/EnergyPlus/DualDuct.hh

```diff
@@ -77,125 +73,84 @@ namespace DualDuct {
         Num
     };
 
-    enum class DualDuctOAMode
-    {
-        Invalid = -1,
-        ConstantOAMode = 11,
-        ScheduleOAMode = 12,
-        DynamicOAMode = 13,
-        Num
-    };
-
     enum class PerPersonMode
     {
         Invalid = -1,
-        ModeNotSet = 20,
-        DCVByCurrentLevel = 21,
-        ByDesignLevel = 22,
```

Comment:
20, 21, 22, I'm sure @amirroth loves this change.

Reply:
@amirroth does!